app/src/main/java/org/sopt/sample/presentation/login/signin/SignInViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package org.sopt.sample.presentation.login.signin
+
+import androidx.lifecycle.*
+import kotlinx.coroutines.launch
+import org.sopt.sample.data.auth.model.SignInRequest
+import org.sopt.sample.data.auth.repository.AuthRepository
+import org.sopt.sample.util.addSourceList
+
+
+class SignInViewModel(private val authRepository: AuthRepository) : ViewModel() {
+
+    val email = MutableLiveData<String>()
+    val pw = MutableLiveData<String>()
+    val isInputValid = MediatorLiveData<Boolean>()
+    private val _signInSuccess = MutableLiveData<Boolean>()
+    val signInSuccess: LiveData<Boolean>
+        get() = _signInSuccess
+
+    init {
+        isInputValid.value = false
+        isEnabledSigninButton()
+    }
+
+    private fun isEnabledSigninButton() {
+        isInputValid.apply {
+            addSourceList(email,pw){inputValidCheck()}
+
+        }
+    }
+
+    private fun inputValidCheck(): Boolean = isEmailValid() && isPwValid()
+
+    private fun isEmailValid(): Boolean = !email.value.isNullOrBlank()
+    private fun isPwValid(): Boolean = !pw.value.isNullOrBlank()
+    private fun getInfo(): SignInRequest {
+        return SignInRequest(email.value.toString(), pw.value.toString())
+    }
```

Comment:
```suggestion
    private val request: SignInRequest
        get() = SignInRequest(email.value.toString(), pw.value.toString())
```

이런건 프로퍼티로 선언하셔도 충분할 것 같네요